README.md

```diff
@@ -23,7 +23,9 @@ This is an example of how you may give instructions on setting up your project l
 
 ## Prerequisites
```

Comment:
To me the pre-requisites section is a section where we add previous steps to achieve what we want to. So to me, apart from this, we should add here:
- How to install everything needed to run the templates (terraform, az cli, etc) (or links/templates so that we ensure the user knows how to install it)